src/SourceBuild/content/build.sh

```diff
@@ -3,13 +3,15 @@
 ### Usage: $0 [options]
 ###
 ### Options:
+###   --ci                         Set when running on CI server
```

Comment:
IIRC, this always got set when building in SB mode. But good to add it anyway.

Reply:
Yes, it gets set for the inner repos but not in the orchestrator itself. I added this so that the minimal console log is used on CI but not locally.